unic/ucd/category/src/category.rs

```diff
@@ -321,41 +325,49 @@ impl GeneralCategory {
 impl GeneralCategory {
     /// `Lu` | `Ll` | `Lt`  (Short form: `LC`)
     pub fn is_cased_letter(&self) -> bool {
+        use self::abbr_names::*;
```

Comment:
I personally think that this specific use case is easier to read with the localized `use abbr_names::*`, because it then mirrors the source almost exactly.

Still, the title of this PR is "Drop `use ...::*` lines outside of data or tests", and this is neither data nor a test, so it either needs to drop the `use ::*` or I would like to see a reasoning somewhere in the text of the PR saying why these get to keep their `use ::*`.